core/src/main/java/org/apache/iceberg/io/ClusteredWriter.java

```diff
@@ -132,6 +132,8 @@ public final R result() {
     return aggregatedResult();
   }
 
+  /** @deprecated will be removed in 1.4.0 */
```

Comment:
Given that 1.3.0 is out, should it be deprecated since 1.4.0, removed in 1.5.0?

Reply:
```suggestion
  /** @deprecated will be removed in 1.5.0 */
```

Reply:
Indeed, this was copy pasted 🤦🏻